assets/src/components/detours/detourRouteSelectionPanel.tsx

```diff
@@ -69,6 +69,7 @@ export const DetourRouteSelectionPanel = ({
                 selectedRouteInfo
               )}
               selectRoutePattern={() => {}}
+              defaultOpened="variants"
```

Comment:
btw, I don't know if you noticed this, but because this only "seeds" the state, and the RPC is programmed to close it's "opened details section" whenever something is selected, this only applies on first render.

Reply:
Hmmm - touché - I'm guessing (although I'll ask) that that's not Kathleen's intent